content/en/blog/falco-0-38-0/index.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Introducing Falco 0.38.0
+date: 2024-05-22 # TODO set the correct date
+author: Luca Guerra
+slug: falco-0-38-0
+tags: ["Falco","Release"]
+---
+
+Dear Falco Community, today we are happy to announce the release of Falco 0.38.0! This is the first Falco release since its [graduation](/blog/falco-graduation/) within the CNCF, and, as usual, brings many improvements and features alongside some pretty big changes in its configuration mechanism.
+
+This release brings an easier to use mechanism to install and configuring your drivers, new rule language features, better support for Falco metrics and many more improvements.
+
+During this release cycle, we merged more than 100 PRs on Falco and more than 180 PRs for libs and drivers, version 0.17.0 and version 7.1.1 respectively. Thank you to our maintainers and contributors, as this would not happen without your support and dedication!
+
+To learn all about these changes, read on!
+
+## What’s new? TL;DR
+
+*Key features:*
+
+* New capabilities in `falcoctl` to automatically select the best driver for your system
+* The Falco configuration file can now be split into multiple files to make it more manageable
+* Rule selection from configuration file or command line
+* Field transformers and value comparison
+* Prometheus metrics support
+* Plugin API enhancements
+
+This release also comes with [breaking changes](#breaking-changes) that you should be aware of before upgrading.
+
+## Major features and improvements
+
+The 0.38.0 release contains a number of feature and UX improvements, here are list of some of the key new capabilities.
+
+### Automatic driver selection ✨
```

Comment:
I'd expand ie: `Driver loader improvements` adding also the ability to automatically fetch kernel headers for `supported-by-driverkit` distros; this means that, in many scenarios, kernel headers are no more mandatory on the nodes.

Reply:
Beautiful <3